tests/conftest.py

```diff
@@ -94,10 +99,23 @@ def do_patch(authorized=True):
             "http://arborist-service/auth/request": {
                 "POST": ({"auth": authorized}, 200)
             },
+            "http://arborist-service/user/requestor_user": {
+                "GET": (
+                    {
+                        "name": "pauline",
+                        "groups": [],
+                        "policies": [{"policy": "test-policy"}],
+                    },
+                    204 if authorized else 403,
+                )
+            },
             "http://arborist-service/user/requestor_user/policy": {
                 "POST": ({}, 204 if authorized else 403)
             },
-            "http://arborist-service/policy?expand": {
+            "http://arborist-service/user/requestor_user/policy/test-policy": {
+                "DELETE": ({}, 204 if authorized else 403)
+            },
+            "http://arborist-service/policy/?expand": {
```

Comment:
Why do we have a trailing `/` after `/policy` and also a leading `?` before `expand`? I thought it was supposed to be a `/policy?expand` as per the document.

Reply:
because gen3authz makes the call with a trailing slash. it doesn't matter, it's the same endpoint and both work